tools/travis/check_python_3.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2018 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
+
+echo '==============================='
+echo 'Running pylint                 '
+echo '==============================='
+
+for filename in $(find lib/rucio bin);
```

Comment:
Maybe only test files named in `changed_files.txt` similar to the check_syntax test.